package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@uniswap/v3-core": "^1.0.0",
+    "@perp/lushan": "^0.11.1",
```

Comment:
This package does not appear to be used in this handler.

Reply:
Will move to devDependencies in package.json